src/renderer_gl.cpp

```diff
@@ -259,14 +271,15 @@ void RendererGL::render(glm::mat4 proj_mat, glm::mat4 view_mat) {
    glUseProgram(programBlur.getId());
 
    // Blur pingpong (N horizontal blurs then N vertical blurs)
-   if (0) {
+
+   if (1) {
       int loop = 0;
       for (int i = 0; i < 2; ++i) {
          if (i == 0)
             glProgramUniform2f(programBlur.getId(), 1, 1, 0);
          else
             glProgramUniform2f(programBlur.getId(), 1, 0, 1);
-         for (int j = 0; j < 100; ++j) {
+         for (int j = 0; j < 1; ++j) {
```

Comment:
why a for up to 1? either remove the for loop or alternatively put the 1 on a variable with a name that explains why is just one

Reply:
Yeah good point, done.